cfme/infrastructure/config_management.py

```diff
@@ -51,10 +51,7 @@ def cfm_mgr_table():
 cfg_btn = partial(tb.select, 'Configuration')
 
 
-def match_page(title):
-    return all((summary_title() == title,
-               browser_title() == 'Red Hat Satellite Provider',
-               controller_name() == 'provider_foreman'))
+match_cur_location = partial(match_location, 'provider_foreman', 'Red Hat Satellite Provider')
```

Comment:
I'll likely specify the exact arguments in my usage, so that its clear what's controller, page title, and summary title.
`partial(match_location, controller='provider_foreman', title='Red Hat Satellite Provider')`